meta/autoid/autoid.go

```diff
@@ -179,10 +183,15 @@ func (alloc *memoryAllocator) Alloc(tableID int64) (int64, error) {
 }
 
 // NewAllocator returns a new auto increment id generator on the store.
-func NewAllocator(store kv.Storage, dbID int64) Allocator {
+func NewAllocator(store kv.Storage, originalDBID, dbID int64) Allocator {
```

Comment:
Maybe we don't need to add an extra argument. If we want to use old schema ID, we just call `NewAllocator(store, oldID)`